oop/src/views/MoveInfoPage.vue

```diff
@@ -108,11 +166,25 @@ export default {
         scrollToError();
         return;
       }
-      pageStateService.setPageIncomplete(routes.MOVE_INFO_PAGE.path)
-      const path = routes.REVIEW_PAGE.path;
-      pageStateService.setPageComplete(path);
-      this.$router.push(path);
-      scrollTo(0);
+
+      this.isLoading = true;
+
+      setTimeout(() => {
+        this.isLoading = false;
+        this.$store.dispatch(formModule + '/' + SET_MOVE_FROM_BC_DATE, this.moveFromBCDate);
+        this.$store.dispatch(formModule + '/' + SET_ARRIVE_DESTINATION_DATE, this.arriveDestinationDate);
+        this.$store.dispatch(formModule + '/' + SET_COUNTRY, this.country);
+        this.$store.dispatch(formModule + '/' + SET_ADDRESS_LINE, this.addressLine);
+        this.$store.dispatch(formModule + '/' + SET_PROVINCE, this.province);
+        this.$store.dispatch(formModule + '/' + SET_CITY, this.city);
+        this.$store.dispatch(formModule + '/' + SET_POSTAL_CODE, this.postalCode);
+
+        pageStateService.setPageIncomplete(routes.MOVE_INFO_PAGE.path)
+        const path = routes.REVIEW_PAGE.path;
+        pageStateService.setPageComplete(path);
```

Comment:
Will need to update the usage of `pageStateService`. Please use `pageStateService.visitPage(path);` instead.